File: quiz/programers/js/level1/소수_찾기.js
function solution(n) {
  const primeNumber = [];
  for (let i = 0; i < n + 1; i++) {
    primeNumber.push(i);
  }

  for (let i = 2; i * i < n; i++) {
    if (primeNumber[i]) {
      for (let j = i * i; j <= n; j += i) {
        primeNumber[j] = 0;
      }
    }
  }

  primeNumber.splice(0, 2);
  const answer = primeNumber.filter(value => value !== 0);
  return answer.length;
}

console.log(solution(10));

File: quiz/programers/python/level1/소수 찾기.py
def solution(n):
    number = [False, False] + [True] * (n - 1)

    for i in range(2, n + 1):
        if number[i]:
            for j in range(2 * i, n + 1, i):
                number[j] = False

    return len(list(filter(lambda x: x, number)))


print(solution(100))